src/prpy/util.py

```diff
@@ -1326,7 +1326,7 @@ def ComputeGeodesicUnitTiming(traj, env=None, alpha=1.0):
     return new_traj
 
 
-def CheckJointLimits(robot, q, deterministic=None):
+def CheckJointLimits(robot, q, hard_limits, deterministic=None):
```

Comment:
@psigen My understanding is that we only use soft limit during planning so when the robot starts at a hard joint limit the planner would through a JointLimitError, but we'd like this to simply plan through instead of erroring out.